commcare_connect/templates/tailwind/pages/opportunity_dashboard/opportunity_resource_modal.html

```diff
@@ -0,0 +1,89 @@
+<div x-show="showModal"
+    x-transition:enter="transition ease-out duration-300"
+    x-transition:enter-start="opacity-0"
+    x-transition:enter-end="opacity-100"
+    x-transition:leave="transition ease-in duration-200"
+    x-transition:leave-start="opacity-100"
+    x-transition:leave-end="opacity-0"
+    class="fixed inset-0 z-50 overflow-y-auto"
```

Comment:
I think you can remove all these x-transition classes etc and use modal-backdrop etc classes. You can checkout existing modals in the ui-main and use same pattern.